test/staging/ArrayBuffer/resizable/access-out-of-bounds-typed-array.js

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-arraybuffer-length
+description: >
+  Automatically ported from AccessOutOfBoundsTypedArray test
+  in V8's mjsunit test typedarray-resizablearraybuffer.js
+features: [resizable-arraybuffer]
+flags: [onlyStrict]
```

Comment:
Out of curiosity, the `features` and `flags` metadata are set by your import script?

Reply:
I wrote the import script so you can provide a command-line option to set the `features` - autodetecting some features might be a nice improvement but I don't think it's exhaustively possible. The script does automatically set `flags: [onlyStrict]` if the file it's converting has a `"use strict"` directive in it.